### PR TITLE
ATAC-Seq update new version of pgt

### PR DESCRIPTION
**Pull request by lldelisle**

### CODE REVIEW

File: topics/epigenetics/tutorials/atac-seq/tutorial.md [hunk 1]
```diff
@@ -194,7 +194,10 @@ The first step is to check the quality of the reads and the presence of the Next
 > > >
 > > >    3. **Overrepresented sequences**
 > > >
-> > >       Nextera adapter sequences are observable in the **Adapter Content** section.
+> > >       One sequence is over represented, it corresponds to the Nextera adapter.
+> > >       It means that you have 306 reads which are exactly the Nextera adapter,
```
Comment (bgruening) [inline topics/epigenetics/tutorials/atac-seq/tutorial.md:198]: This sentence reads a little bit repetitive. Maybe less "Nextera adapters"

**Comment by lldelisle** [comment]
I will follow Bjorn advice. Wait a minute.

**Comment by hrhotz** [comment]
ok, sure

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**